CHANGELOG.md

```diff
@@ -18,46 +18,45 @@
 - Add support for "use-octavia" setting in Openstack provider specs. It defaults to "true" but leaves the possibility to set it to "false" if your provider doesn't support Octavia yet but Neutron LBaaSv2
  [#6529](https://github.com/kubermatic/kubermatic/issues/6529) 
 - Add components override field to set nodeportrange for apiserver [#6533](https://github.com/kubermatic/kubermatic/issues/6533) 
-- Overhaul CA handling, allow to configure a global CA bundle for every component. The OIDC CA ile has been removed, manual updates can be necessary. [#6538](https://github.com/kubermatic/kubermatic/issues/6538) 
```

Comment:
Why are we removing this change from the changelog? It also has `action required` in the release note, which we should highlight.